resources/openapi.yaml

```
@@ -51,22 +50,33 @@ components:
             - gurmand
             - sports
           type: string
-        image-url:
+        imageURL:
           type: string
+        createdAt:
+          type: integer
+          format: int64
+          minimum: 0
+        updatedAt:
+          type: integer
+          format: int64
+          minimum: 0
       required:
         - id
         - name
         - details
         - category
-        - image-url
+        - imageURL
+        - createdAt
+        - updatedAt
       description: community information
-      title: Community
       example:
         id: f61f5f38-174b-43e1-8873-4f7cdbee1c18
         name: 辛いものの部
         details: 辛いものが好きな人集まれー
         category: gurmand
-        image-url: https://picsum.photos/id/292/{width}/{height}.jpg
+        imageURL: https://picsum.photos/id/292/{width}/{height}.jpg
+        createdAt: 1647307406
+        updatedAt: 1647307406
```

```suggestion
        imageURL: https://picsum.photos/id/292/{width}/{height}.jpg
        createdAt: 1647307406
        updatedAt: 1647307406
        joined: true,
        joinedMembers: []
```

joinedMembers には 自分自身を含めたユーザーの配列が入っていると嬉しい
実装めんどくさかったらとりあえず空配列だけ返しておいて後から実装でも大丈夫🙏

---

自身を含めた全員のメンバーを出そう、はサービス考えたときに現実的に無茶かなと思います。（1ページに表示するコミュニティ数に一定の制限をかけるか、コミュニティがどのくらいの人数を想定するのかによりますが）

---

なるほど、もし実現するとしたら所属ユーザー一覧を取得する API を別途用意するべきか（やるかどうかはまた別）

---

community を与えるとメンバー一覧が返ってくる、は普通なのかなと思いますね
あと一般的なのは、 community のメンバー数とか (いわゆるフォロワーみたいな感じ、これのほうが必要なんじゃないかなと)

---

あー！なるほど！メンバー数ね！それいい

---

`joined: boolean;` `membership: int;` らへんが良さそう

---

https://github.com/MokkeMeguru/spa_guide_sns_server/pull/23/commits/4a78a4158a559f85288ea730a9869d5380a6638d 
:done:

isJoined (null | boolean) にしちゃったけどどうだろうか